kubetest/local.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"time"
+)
+
+type localCluster struct{}
+
+var _ deployer = localCluster{}
+
+func newLocalCluster() *localCluster {
+	return &localCluster{}
+}
+
+func (n localCluster) Up() error {
+	script := "./hack/local-up-cluster.sh"
+	cmd := exec.Command(script)
+	cmd.Env = os.Environ()
+	cmd.Env = append(cmd.Env, fmt.Sprintf("ENABLE_DAEMON=true"))
+	return finishRunning(cmd)
+}
+
+func (n localCluster) IsUp() error {
+	log.Print("Noop - LocalCluster IsUp()")
```

Comment:
here probably should not be noop? Add something checks if the local cluster is up or not?

Reply:
yep, fixing. will check using kubectl get nodes